gui.py

```diff
@@ -17,25 +20,44 @@ def main():
     ascii_img = handle_image_conversion(image_file_path)
     print(ascii_img)
     write_to_txtfile(ascii_img, "output.txt")
+    success.config(
+        text='Image converted successfully! File saved as \'output.txt\'', fg='white')  # Success message displays upon conversion
 
 
 if __name__ == '__main__':
-    root = Tk()
-    root.geometry("800x220")
+    root = tk.Tk()
+    root.geometry("900x500")
     root.title("Image to ASCII converter")
     root.configure(background="black")
     root.resizable(0, 0)
 
-    Label(root, text="Image to ASCII Convertor", fg="black", font=("Times", 25, "bold"), width=25).pack()
+    def browsePic():  # Allows user to browse directories for file and provides path to entry1
+        global entry1
+        tempdir = filedialog.askopenfilename(parent=root, filetypes=(
+            ('png files', '*.png'), ('jpeg files', '*.jpg'), ('all files', '*.*')), title='Please select a file')
+        entry1.insert(tk.END, tempdir)
+
+    Label(root, text="Image to ASCII Converter", fg="black",
+          font=("Times", 25, "bold"), width=25, pady=5, padx=25).pack()
 
-    label1 = Label(root, text="Please enter file path - ", fg="black", font=(15))
-    label1.place(x=80, y=60)
+    label1 = Label(root, text="Please enter the file path or choose a file:",
+                   fg="white", bg='black', font=(20))
+    label1.place(x=140, y=100)
 
-    entry1 = Entry(root, bd=5, font=(15), width=70)
+    entry1 = Entry(root, bd=5, font=(15), width=60)
     entry1.bind(main)
-    entry1.place(x=80, y=100)
+    entry1.place(x=140, y=140)
+
+    browse_btn = tk.Button(root, text='Choose File', font=("Arial", 16), pady=6,
+                           width=7, bd=-2, command=browsePic)  # Button to browse through directories
+    browse_btn.place(x=705, y=140)
+
+    convrt_btn = Button(root, text="Convert", font=("Arial", 20),
+                        width=10, padx=5, pady=5, command=main)
+    convrt_btn.place(x=340, y=180)
 
-    button1 = Button(root, text="Convert", font=("Times", 20), width=10, padx=5, pady=5, command=main)
-    button1.place(x=310, y=150)
+    success = Label(root, text='', bg='black')
+    success.pack(pady=5)
+    success.place(x=260, y=220)
 
```

Comment:
- Try on working for saving output.txt as optional ' Add a button for it'

Reply:
Isn't that what the 'Convert' button does already?

Ideally one would change the name of the 'Convert' button to 'Save as txt file' and create a way to display the ascii conversion in the GUI itself. Or is that not possible?

Thanks

Reply:
- Convert button converts and prints ascii art as stdout
- According to #12, Output file should be optional, not default

Reply:
- For the purpose of displaying the ascii art in gui, you can save the file. NP

Reply:
- stop saving output as default for now. I'll merge this pr, you can work on gui display and submit another pr

Reply:
Thanks! Already working on it. In the meantime, I update the width.